### PR TITLE
62 fix the empty links on header

### PR DESCRIPTION
**Pull request by Eric-ZheranLi**
1. Fix the empty link for statistics page in header's nav bar.

2. Implement the statistics page as an exercise for myself.
image
image

### CODE REVIEW

File: src/components/StatisticsContent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+const StatisticsContent = () => {
+  /* focus = 'day' leading to 'For the day' page, 'all the time' to 'For all the time' page*/
+  const [focus, setFocus] = useState('day');
```
Comment (tatoMa) [inline src/components/StatisticsContent/index.tsx:6]: Think about the naming convention again.

File: src/components/StatisticsContent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    all_correct_answers: 43,
+    sprint_words_studied: 5,
+    sprint_correct_answers: 90,
+    sprint_best_series: 7,
+    'audio-call_words_studied': 5,
+    'audio-call_correct_answers': 90,
+    'audio-call_best_series': 7,
+  };
+
+  const page_data = focus === 'day' ? today_statistics : all_time_statistics;
```
Comment (tatoMa) [inline src/components/StatisticsContent/index.tsx:30]: Think about the naming convention again.

File: src/layout/Header/index.tsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ const Header = () => {
             <li>
               <a
                 title="Statistics"
-                href="#"
+                href="/statistics-content"
                 className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
               >
```
[the remark this answers is not in the document]
Comment (Eric-ZheranLi) [reply src/layout/Header/index.tsx]: https://github.com/taodemy/taotalk/blob/9ef86ce2e82410b09c9859df058aa6cc23c1f49a/src/layout/Header/index.tsx#L25
https://github.com/taodemy/taotalk/blob/9ef86ce2e82410b09c9859df058aa6cc23c1f49a/src/layout/Header/index.tsx#L30-L36
https://github.com/taodemy/taotalk/blob/9ef86ce2e82410b09c9859df058aa6cc23c1f49a/src/layout/Header/index.tsx#L39-L45
https://github.com/taodemy/taotalk/blob/9ef86ce2e82410b09c9859df058aa6cc23c1f49a/src/layout/Header/index.tsx#L52-L59

Corresponding \<a\> tags have been replaced by \<Link\> tags.

File: src/components/StatisticsContent/index.tsx (hunk 1 above)
Comment (Eric-ZheranLi) [reply src/components/StatisticsContent/index.tsx]: The statistics content pages have been removed from this issue.

**Review by LULIU-9 — APPROVED** [review]
LGTM

**Pull request merged**